bim2sim/plugins/PluginTEASER/bim2sim_teaser/task/create_result_df.py

```diff
@@ -190,14 +190,13 @@ def map_zonal_results(bim2sim_teaser_mapping_selected, elements: dict):
         """
         bim2sim_teaser_mapping = {}
         space_guid_list = []
+        all_tzs = filter_elements(elements, 'ThermalZone')
         agg_tzs = filter_elements(elements, 'AggregatedThermalZone')
-        if agg_tzs:
-            for agg_tz in agg_tzs:
-                space_guid_list.append(agg_tz.guid)
-        else:
-            tzs = filter_elements(elements, 'ThermalZone')
-            for tz in tzs:
-                space_guid_list.append(tz.guid)
+        normal_tzs = list(set(all_tzs) - set(agg_tzs))
+        for agg_tz in agg_tzs:
+            space_guid_list.append(agg_tz.guid)
+        for tz in normal_tzs:
+            space_guid_list.append(tz.guid)
```

Comment:
Why don't you just directly append all guids from all TZs in all_tzs, as both (aggregated and common) TZ elements have the attribute guide? 
This seems to be redundant.

Reply:
you are right. Previously I had in mind to get the zone guids of all aggregated zones, but for now I use only the new guid of aggregated zone, so this was redundant. I fixed it.